themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -12,6 +14,13 @@ addEventListener("beforeunload", () => {
   }
 });
 
+setTimeout(function() {
```

Comment:
For anonymous functions we prefer to use the arrow syntax as a stylistic choice, but this could also be a named function if needed.

```suggestion
setTimeout(() => {
```